### DISCUSSION TITLE
Using wiki.js with a screen reader

### DISCUSSION THREAD

**Discussion by philippkraft** [requarks/wiki#6482] [Help / Questions]
Is someone using wiki.js with a screen reader? Our first try shows, that the drop down menu for page actions seems not accessible from the screen reader. It is possible to open the menu (tab+space) but I can't select any menu item with the keyboard.

Background: we are currently testing wiki.js and it looks great, but one colleague has impaired visibility and uses a screen reader. 

Host: Ubuntu 22.04, Docker 20.10, wiki.js 2.5.298

**Comment by NGPixel** [comment]
v2 has a few shortcomings in regard to accessibility, you just listed one of them. Accessibility is being greatly improved in the upcoming v3.

**Reply by philippkraft** [reply]
Great to hear. Do you have some rough timetable regarding v3? Perhaps we can help with beta tests regarding the accessibility?
